src/App.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <div id="app">
-    <Navbar />
-    <router-view/>
+    <Header />
+    <Home />
```

Comment:
Moved <router-view> to inside the StudentFeedback component, because that's the target of the click event when someone clicks on a module.

Reply:
Sounds good.